types/selenium-webdriver/bidi/index.d.ts

```diff
@@ -0,0 +1,95 @@
+import { EventEmitter } from "events";
+
+export interface BuildInfo {
+    version: string;
+}
+
+export interface OsInfo {
+    arch: string;
+    name: string;
+    version: string;
+}
+
+export interface ResultData {
+    build: BuildInfo;
+    message: string;
+    os: OsInfo;
+    ready: boolean;
+}
+
+export interface StatusResult {
+    id: number;
+    result: ResultData;
+    type: string;
+}
+
+export interface Params {
+    method: string;
+    params: {
+        [key: string]: any;
+    };
+}
+
+export class Index extends EventEmitter {
+    id: boolean;
+    connected: boolean;
+    events: string[];
+    browsingContexts: string[];
+
+    /**
+     * Create a new websocket connection
+     * @param _webSocketUrl
+     */
+    constructor(_webSocketUrl: string);
+
+    /**
+     * @returns {boolean}
+     */
+    get socket(): WebSocket;
+
+    /**
+     * @returns {boolean}
+     */
+    get isConnected(): boolean;
+
+    /**
+     * Get Bidi Status
+     * @returns {Promise<any>}
+     */
+    get status(): Promise<StatusResult>;
```

Comment:
This seems like it should return the same type as `send()`? Where does `StatusResult` come from? https://www.selenium.dev/selenium/docs/api/javascript/bidi_index.js.html#line58

Reply:
Thanks for clarifying! Can you add tests for `StatusResult`?